drools-wb-services/drools-wb-verifier/drools-wb-verifier-web-worker/src/main/resources/org/drools/workbench/verifier/webworker/linker/ScriptTemplate.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+$stats = function() {
```

Comment:
I assume this is a template to be used by custom linkers?

Sometimes it's nice to put some comments explaining things; so when I have to fix this when you're on holiday I have some clue ;-)